frontend/app/routes/protected/person-case/state-machine-models.ts

```diff
@@ -127,6 +128,7 @@ export type InPersonSinApplication = {
 declare module 'express-session' {
   interface SessionData {
     inPersonSinApplications: Record<string, SnapshotFrom<Machine> | undefined>;
+    createdCases: Record<string, PersonSinCase>;
```

Comment:
I think this should go in the global augmentation file (`~/@types/express-session.d.ts`) since it is used outside of `person-case`.